cli/src/commands/git/push.rs

```diff
@@ -140,6 +140,9 @@ pub struct GitPushArgs {
     /// Only display what will change on the remote
     #[arg(long)]
     dry_run: bool,
+    /// Display per-commit reasons for failed pushes
+    #[arg(long)]
+    explain_denied: bool,
```

Comment:
Feedback about the user-facing phrasing is especially welcome, I don't have any real attachment to this one. Same with all naming, really.